tests/asgi/test_async.py

```diff
@@ -1,26 +1,33 @@
-import typing
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Optional
 
 import pytest
-from starlette.testclient import TestClient
 
 import strawberry
-from strawberry.asgi import GraphQL
+
+if TYPE_CHECKING:
+    from starlette.testclient import TestClient
 
 
 @pytest.fixture
 def test_client() -> TestClient:
+    from starlette.testclient import TestClient
+
+    from strawberry.asgi import GraphQL
+
     @strawberry.type
     class Query:
         @strawberry.field
-        async def hello(self, name: typing.Optional[str] = None) -> str:
+        async def hello(self, name: Optional[str] = None) -> str:
             return f"Hello {name or 'world'}"
 
     async_schema = strawberry.Schema(Query)
-    app = GraphQL(async_schema)
+    app = GraphQL[None, None](async_schema)
```

Comment:
Interesting is this for proper typing?

Reply:
yup! it's unfortunate we can't have default typevars :(

Reply:
yup! it's unfortunate we can't have default typevars :(